scripts/metadata_comparison/README.MD

```diff
@@ -0,0 +1,36 @@
+# Metadata Comparison Scripts
+
+This `metadata_comparison` python project provides tools to compare workflows run
+in different Cromwell environments for cost and performance. 
+
+## Running a script
+
+Choose a script to run. For this example we will use the `extractor`.
+
+From this top-level directory `metadata_comparison` directory (ie the one 
+containing this README.MD file), run:
+
+```sh
+# python3 -m metadata_comparison.extractor <ARGS>
+```
+
+### Questions
+
+- Q: Why not run the scripts directly, eg `python3 extractor.py`?
+  - A: Running python from this outer directory allows it to discover the `metadata_comparison` 
+  project, and thus allows imports across and between scripts.
+
+## Unit tests
+
+To run the python unit tests from the top-level `metadata_comparison` directory 
+(ie the one containing this README.MD file), run:
+```sh
+# python3 -m unittest discover -v
+```
```

Comment:
Same comment as above. I'm open to refactoring the script UX some day, but it's standard python practice AFAICT, and would probably be fodder for a part 5 or 6 rather than an MVP part 1